<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>131</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>233</width>
    <height>131</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScanWatcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="HL_INPUT">
     <item>
      <layout class="QVBoxLayout" name="VL_LABEL">
       <item>
        <widget class="QLabel" name="lHost">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lPort">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VL_LINEEDIT">
       <item>
        <widget class="QLineEdit" name="leHost">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
         <property name="placeholderText">
          <string>192.168.0.123</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePort">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="inputMask">
          <string>00009</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
         <property name="placeholderText">
          <string>25015</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HL_BUTTON">
     <item>
      <spacer name="HS_BUTTON">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bConnect">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
